File: pattern.java
import java.util.Scanner;

public class pattern {
    public static void main(String[] args) {
        System.out.println("Enter a number: ");
        Scanner sc = new Scanner(System.in);
        int number = sc.nextInt();
        sc.close();

        //Increasing Loop
        for (int i = 0; i < number; i++){
            for (int j = 0; j < i; j++){
                System.out.print("* ");
            }
            System.out.println("");
        }

        // Decreasing Loop
        for(int i = number; i > 0; i--){
            for(int j = i; j > 0; j--){
                System.out.print("* ");
            }
            System.out.println("");
        }
    }
}

File: pattern.py
def pattern(n):
    #increasing loop
    for i in range(0, n, +1):
        for j in range(0, i):
            print("* ", end="")
        print("\r")

    #decreasing loop
    for i in range(n, 0, -1):
        for j in range(0, i):
            print("* ", end="")
        print("\r")

number = int(input("Enter a number: "))
pattern(number)